samples/authenticateExplicit.js

```diff
@@ -0,0 +1,73 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Lists storage buckets by authenticating with ADC.
+ */
+function main() {
+  // [START auth_cloud_explicit_adc]
+  /**
+   * TODO(developer):
+   *  1. Set up ADC as described in https://cloud.google.com/docs/authentication/external/set-up-adc
```

Comment:
Should this link be https://cloud.google.com/docs/authentication/provide-credentials-adc?
```suggestion
   *  1. Set up ADC as described in https://cloud.google.com/docs/authentication/provide-credentials-adc
```

Reply:
Resolving via @piaxc's note